patches/contrib/ui_patterns_views-preview.patch

```diff
@@ -96,7 +96,7 @@ index 388b287..6f835d4 100644
 +  foreach ($view->field as $id => $field) {
 +    // render this even if set to exclude so it can be used elsewhere.
 +    $field_output = $view->style_plugin->getField($row->index, $id);
-+    $empty = $field->isValueEmpty($field->getValue($row), $field->options['empty_zero']);
++    $empty = $field->getPluginId() == 'field' && $field->isValueEmpty($field->getValue($row), $field->options['empty_zero']);
```

Comment:
👍

Reply:
this works to fix an issue I was seeing when we mapped "Link to Content" to other fields in a pattern view.

previously, the field for link to content would always render as null, even when there was a node detail page to link to.

<img width="851" alt="Screen Shot 2020-06-01 at 5 56 06 PM" src="https://user-images.githubusercontent.com/12848168/83463099-0446dc80-a43b-11ea-9197-d0fd75bb4e57.png">